specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2021-03-01/AppServiceCertificateOrders.json

```diff
@@ -1163,7 +1163,11 @@
                   "ExpirationNotInRenewalTimeRange",
                   "SubscriptionNotActive"
                 ],
-                "type": "string"
+                "type": "string",
+                "x-ms-enum": {
+                  "name": "ResourceNotRenewableReason",
+                  "modelAsString": false
```

Comment:
the `modelAsString=false` seems not same as latest 2022-03-01, which is `true`. Would you double check?

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2022-03-01/AppServiceCertificateOrders.json#L1268

Reply:
Thanks. Let me correct it in the PR.

Reply:
@weidongxu-microsoft Thanks for the comment. This has been addressed.